src-tauri/Cargo.toml

```diff
@@ -1,39 +1,59 @@
 [package]
 name = "subspace-desktop"
-version = "0.5.2"
+version = "0.5.3"
 description = "Subspace desktop"
 authors = ["Subspace Labs <https://subspace.network>"]
 license = "Apache-2.0"
 repository = "https://github.com/subspace/subspace-desktop"
 edition = "2021"
 
 [build-dependencies]
-substrate-build-script-utils = { version = "3.0.0", git = "https://github.com/paritytech/substrate", rev = "5d3e7c4ee9f5c8e370022d5f3fa9723185710831" }
+substrate-build-script-utils = { version = "3.0.0", git = "https://github.com/paritytech/substrate", rev = "24bea4c3cba7479e2cf2976a21e8111dfda6b1cd" }
```

Comment:
We switched to our fork in https://github.com/subspace/subspace/pull/533 (same commit though) to make builds faster and use less disk space

Reply:
this is wonderful 🥳